README.md

```diff
@@ -192,6 +192,12 @@ While the aforementioned example should work for training, we leverage the
 cocoApi for computing the accuracies during testing. Thus, test datasets
 should currently follow the cocoApi for now.
 
+## Finetuning from Detectron weights on custom datasets
+Create a script `tools/trim_detectron_model.py` like [here](https://gist.github.com/wangg12/aea194aa6ab6a4de088f14ee193fd968).
+You can decide which keys to be removed and which keys to be kept by modifying the script.
+
+Then you can simply point the converted model path in the config file by changing `MODEL.WEIGHT`.
```

Comment:
Can you also point to the relevant issue?
Something like

> For further information, please refer to https://github.com/facebookresearch/maskrcnn-benchmark/issues/15

Reply:
Sure.